### ISSUE TITLE
Input text field controls cover RTL characters

### ISSUE THREAD

**Issue by kontur** [justvanrossum/fontgoggles#126]
When typing in a RTF string the cogwheel for loading a text file will collide with the string in the text input:

<img width="1324" alt="Screenshot 2020-10-21 at 13 52 52" src="https://user-images.githubusercontent.com/1893577/96710509-da1f9800-13a4-11eb-942e-9f986c053522.png">

The UI element for the text input should probably end before the options dropdown and up down arrow icon.

**Comment by justvanrossum** [comment]
Hm, I can't reproduce this, and this indeed should not happen. Does this happen when you start FG with a fresh window?

On which OS are you, and can you give me precise steps how to reproduce?

(I know it's possible mess up the UI state in some cases when resizing the various panes in extreme ways, but it doesn't look like that's what happened here.)

**Comment by kontur** [comment]
Ugh, starting Fontgoggles today I thought I was just a glitch. I usually have Fontgoggles half my window width and end up playing around with showing/hiding the left two UI columns and resizing quite a bit, and that's how I figured out what caused it.

Start out by typing RTL text in the input.

<img width="1059" alt="Screenshot 2020-10-22 at 9 59 07" src="https://user-images.githubusercontent.com/1893577/96836412-9e8cd880-144d-11eb-8a45-2f6489a28725.png">

Resize not the window, but the left most column to push the text input to not fit within its center column.

<img width="1060" alt="Screenshot 2020-10-22 at 9 59 21" src="https://user-images.githubusercontent.com/1893577/96836443-a9476d80-144d-11eb-9963-a071d412b350.png">

Then make the first column "normal" size again or hide the left columns so that the text input can reappear, and voilà, it is now mangled.

<img width="1059" alt="Screenshot 2020-10-22 at 9 59 35" src="https://user-images.githubusercontent.com/1893577/96836508-bcf2d400-144d-11eb-915c-a1c438279ac9.png">

Seeing the steps to reproduce this it seems to be the edge case of an edge case in the UI library, so I don't expect a fix for this; I was just baffled yesterday when I couldn't make it go away with resizing :)

**Comment by justvanrossum** [comment]
Yeah, this is indeed rooted in a problem with the "vanilla" UI library. I tried to make it hard for users to make columns "too narrow", but you've found a good way to do it anyway...

It'll be hard to fix indeed, and for now I would simply advice, regarding your second step: "don't do that"... Sorry for the inconvenience.

**Comment by kontur** [comment]
No worries. After I figured out how to make this happen I would say it's not a likely case 👍
